Source/Csla/Core/BusinessBase.cs

```diff
@@ -3048,10 +3048,10 @@ protected void LoadPropertyAsync<R>(PropertyInfo<R> property, Task<R> factory)
     #endregion
 
     #region IsBusy / IsIdle
-
-    [NonSerialized]
-    [NotUndoable]
-    private bool _isBusy;
+    /// <summary>
+    /// Indicates object is busy or not 
+    /// </summary>
+    private int _isBusyCounter;
```

Comment:
The old attributes
```csharp
[NonSerialized]
[NotUndoable]
```
are still missing.

Reply:
Done